### ISSUE TITLE
NetCat Listen not working

### ISSUE THREAD

**Issue by KsAmJ** [frwololo/PS4_PS5-ESP8266-Server#3]
NetCat Listen not working after compiling it and load it on board
is there a library needs to be added to Arduino?
I have ESP 8266 ESP-12E

**Comment by frwololo** [comment]
Not that I know of. If it compiles, it "should work"....does the released
binary I compiled work for you ?

**Comment by KsAmJ** [comment]
Yes but it takes some time to send to netcat payload like the netcat gui
hang for 30 sec.


Also I wanted to update the web assets to use the new kstuff for 3.20 which
always a pain to wait a binary build releases. I wish there is an easy way
to update the host files and be good to go rather than waiting for the
boards bins

**Comment by frwololo** [comment]
You should be able to access the admin section on the device to replace all
the files. You don't need to wait for a new binary, I do not plan to create
a new binary. Check the Readme on how to replace the exploit files.
